opta/commands/secret.py

```diff
@@ -38,15 +42,14 @@ def view(secret: str, env: Optional[str], config: str) -> None:
     _raise_if_no_k8s_cluster_exists(layer)
 
     configure_kubectl(layer)
-    load_kube_config()
-    v1 = CoreV1Api()
-    api_response = v1.read_namespaced_secret("secret", layer.name)
-    if secret not in api_response.data:
+    create_namespace(layer.name)
+    secrets = deep_merge(get_manual_secrets(layer.name), get_linked_secrets(layer.name))
+    if secret not in secrets:
```

Comment:
So this can happen in multiple situations:
1) You haven't run an apply yet - so the linked secrets are not available. Solution here is to run apply
2) You haven't set a manual secret using the cli. Solution here is to run `secret update`
3) You don't have this manual secret in the yaml.

I think we should print the appropriate message in each of the cases. Instead of just saying "You sure you set it in your yaml".

Reply:
`You haven't run an apply yet - so the linked secrets are not available. Solution here is to run apply` -- I'm afraid that there's no way of telling what is a valid linked secret and what is not. Sure, we can try and parse the yaml again and see all the manual secrets, but we won't know if a given string is a valid linked secret or just baloney.

Going over whether this is a manual secret or not is also a tad tedious-- I can reform this to give an informative error message such as "You specified to see secret X but currently we only have secrets [Y ,Z]-- please use the opta secret update command to add the new secrets!"

Reply:
Ah. Let's do this instead: `We couldn't find a secret named <xyz>. You either need to add it to your opta.yml file or if it's already there - update it via secret update.`